src/store/Harvesting.ts

```diff
@@ -152,14 +156,25 @@ export default {
             commit('setPollingTrials', pollingTrials);
         },
         async FETCH_STATUS({ commit, rootGetters, dispatch }, node?: [string, NodeModel]) {
-            const currentSignerAccountInfo: AccountInfo = rootGetters['account/currentSignerAccountInfo'];
+            const csAccountInfo: AccountInfo = rootGetters['account/currentSignerAccountInfo'];
```

Comment:
i'd suggest not shortening and using original name

Reply:
original name is being used in the line 168(and in the rest of the function), that's why I changed here. In line 168 we are refreshing the current signer account info object by a remote call. Previously I tried to keep the name `currentSignerAccountInfo`  in line 159 and call the new constant `refreshedCurrentSignerAccountInfo`, but didn't feel right when I looked at it so I decided to go with this version. I'd like to hear if you have any suggestions on naming here?

Reply:
naming is fine, if there's line limit hit, can't it be split in half like
```ts
const currentSignerAccountInfo: AccountInfo =
[tab][tab]rootGetters['account/currentSignerAccountInfo'];
```

Reply:
thanks @gimre-xymcity, there wasn't any line limit issue but refactored to an Address type instead as it will clear the confusion and fit better 👍